File: redcap_v15.7.6/API/examples/r/export_instruments_pdf.R
#!/usr/bin/env Rscript

source('config.R')
library(RCurl)

result <- postForm(
    api_url,
    token=api_token,
    content='pdf',
    returnFormat='json',
    binary=TRUE
)

f <- file('/tmp/export.pdf', 'wb')
writeBin(as.vector(result), f)
close(f)

File: html/redcap_v15.7.6/API/examples/perl/export_instruments_pdf.pl
#!/usr/bin/env perl

use strict;
use warnings;
use LWP::Curl;

my %config = do 'config.pl';

my $fields = {
    token   => $config{api_token},
    content => 'pdf',
    format  => 'json'
};

my $ch = LWP::Curl->new(auto_encode => 0);
my $content = $ch->post($config{api_url}, $fields, $config{referer});

open(OUTFILE, '>', '/tmp/export.pdf');
print OUTFILE $content;
close(OUTFILE);
